process.md

```diff
@@ -0,0 +1,108 @@
+
+- Parachains
+- Wallets
+- API Builders
+- Validators
+- Custodians
+
+**2. Scope of Work**
+
+**2.1 Documentation**
+
+- **Internal Processes** : Documenting standard operating procedures, workflows, and best practices for the team.
+- **Basic Tutorials** : Creating guides for new users and stakeholders to engage with Paseo Network.
+- **Design Decision Documentation** : Recording rationales behind significant design decisions for transparency and future reference.
+- **Parameter Change Documentation:** Keeping track of any changes made to the network's parameters, ensuring they're well-documented and justified.
+
+**2.2 Coordination Meetings**
+
+- **Weekly Internal Report every Thursday** : A routine meeting to discuss progress, challenges, and plan for the upcoming week. The meeting must be documented in a Google Docs Shared where all the team can be aligned regarding the insights from the team.
+
+**2.3 Community Management**
+
+- **Bridge Management** (_Platforms: Element, Discord - TBD_): Choosing and maintaining communication platforms for effective interaction with the community.
+- **Paseo's GitHub Organization Maintenance** : Overseeing the GitHub repository, ensuring it's up-to-date and well-organised.
+- **Element Channels** : Managing various channels with different topics, such as:
+  - **Public** : Public channel to share every update and news to the complete community
+  - **Tech** : Developer coordination channel and other Support channel for efficient communication and collaboration between them.
+  - **Management Channels** : Specialised channels for internal and core team discussions.
+
+**2.4 Social Media Management**
+
+- **X Account** : Engaging with the community, sharing updates, and announcements.
+- Other Social Media Updates (_Weekly. Optional_): Regular updates across different platforms to keep the community informed.
+
+**3. Monitoring**
+
+- Validator Monitoring (Service Level Agreements): Ensuring validators operate within agreed-upon standards and parameters.
+- Bot Development:
+  - Large Balance Monitoring: Keeping track of significant balance changes in the network.
+  - OpenGov Monitoring: Monitoring governance activities and proposals.
+
+**4. Runtime Management**
+
+- **Track, Add, Remove Pallets** : Managing the components of the runtime environment to ensure optimal performance.
+- **Runtime Upgrades** : Establishing a timeline (TBD) and process for upgrading the network.
+- **Pallet Configuration Analysis** : Assessing whether new or existing pallet configurations align with Paseo's requirements.
+
+**5. Runtime Upgrade Testing and Coordination**
+
+- **Local Dry-Runs with Live State** : Testing upgrades in a controlled environment to ensure stability.
+- **Backporting** : Applying new features or fixes from the mainnet to the testnet.
+- **Bricked Parachains Resolution** : Developing strategies to deal with malfunctioning parachains.
```

Comment:
This one gets tricky.

I am not saying we should not help teams struggling with their paras, but while it is easier for us to provide some guidelines for more generic scenarios, this kind of problems usually emerge from cases very specific ti each chain.

I don't know if setting this precedent is the best thing.

Reply:
```suggestion
- **Bricked Parachains Resolution** : In case of malfunctioning parachains the team will provide the corresponding support form the relay chain perspective. Below there are more details about support tasks scope:

Support tasks that are within the scope:

- Force Storage Items for parachains (including runtime storage item)
- Remove parachain locks
- Dmp queues that usually come from forcing wasm changes force the parachains
- Force lifecycle upgrades
- Top up balances for sovereign accounts

Support tasks that are **not** in the scope:

- Analyze parachain failed storage migrations.
- Check parachain configuration.
- Check parachain node logs.
```